api/polling/README.md

```diff
@@ -26,11 +26,12 @@ Set up a scheduled process to fetch all orders or shipments that have been creat
 #### Polling best practices
 
 - Use a consistent polling interval (typically 5 minutes)
-- Include the `lastUpdatedAt` from previous poll response as the `lastUpdatedAfter` filter
-- Set `limit=100` (maximum allowed value) to control response size
+- Include the `lastUpdatedAt` timestamp from previous poll response as the `lastUpdatedAfter` filter.
+  - **Do not use the current time** (system clock) for `lastUpdatedAfter` – clock skew or availability issues can cause you to miss updates.
+- Set `limit=100` (the maximum allowed value) to control response size.
 - Handle pagination when needed:
-  - If response contains `total` > 100, not all updates were returned
-  - Either decrease the polling period (recommended) or use `offset` parameter for paging
+  - If response contains `total` > 100, not all updates were returned.
+  - Either decrease the polling period (recommended) or use `offset` parameter for paging.
```

Comment:
I remember I was doubting about this bullet before. I think we should advise both: support for pagination (just in case), and as a good practice: decrease polling period